bin/mozilla/gl.pl

```diff
@@ -1238,9 +1238,9 @@ sub gl_subtotal {
 sub transactions_to_csv {
 
     $filename = 'gl';
-    my $name;
-    do { $name = tmpnam() } until $fh = IO::File->new( $name, O_RDWR | O_CREAT | O_EXCL );
-    open( CSVFILE, ">$name" ) || $form->error('Cannot create csv file');
+
+    my ($fh, $name) = tempfile();
+    close $fh;
```

Comment:
I got the error:

Error!
Cannot close csv file

Reply:
Let me see why it is like this and then I update.

Reply:
ok :) btw. why is ehre a close but not at the other places?